### ISSUE TITLE
Watch does not reprocess eleventy.js

### ISSUE THREAD

**Issue by pamtbaau** [11ty/eleventy#340]
Windows 10
Node 11.4.0

Downloaded starter 'eleventy-base-blog'

It seems that `npm run watch` does not process changes in `eleventy.js`. Although changes in `eleventy.js` are being detected and files are being written to '_site'...

`npm run build` does process any change into the site.

**Comment by jevets** [comment]
Looks like `watch` does watch `.eleventy.js` in version 0.6.0. 

Maybe it's time to bump up the base blog dependency.

**Comment by pamtbaau** [comment]
Forgot to mention that I upgraded all npm dependencies to the latest versions.
```
  "devDependencies": {
    "@11ty/eleventy": "^0.6.0",
    "markdown-it": "^8.4.2",
    "markdown-it-anchor": "^5.0.2"
  }
```

**Comment by jevets** [comment]
Hmmm, wonder if it's a Windows specific issue? It watches `.eleventy.js` and reloads for me on Mac on Node 10.14.x and Eleventy 0.6.0.

Probably need @zachleat on this one

Same thing happens with `eleventy --serve`? 
Same thing happens when running `npx eleventy --watch` directly (instead of `npm run watch`)?

**Comment by pamtbaau** [comment]
Will do some more tests and get back to you....

**Comment by pamtbaau** [comment]
Same thing happens with `eleventy --serve`?   YES
Same thing happens when running `npx eleventy --watch` directly (instead of npm run watch)? YES

**Comment by pamtbaau** [comment]
@zachleat This is what I did:
- Downloaded eleventy-base-blog
- Upgrade all npm packages
- npm install
- npm run watch

Website looks as expected. It shows 3 latest posts at bottom.

- Changed `eleveny.js`, by setting `n` hardcoded:
```
  // Get the first `n` elements of a collection.
  eleventyConfig.addFilter("head", (array, n) => {
    n = 1;                                    <-- Changed the number of posts to return
    if( n < 0 ) {
      return array.slice(n);
    }

    return array.slice(0, n);
  });
```

The change triggers the `watch` and a new site is being generated.
- But, it still shows 3 latest posts at bottom....

When `npm run build` is run, the number of posts shown is correct.
When restarting `npm run watch`, the number of posts shown is correct.

**Comment by zachleat** [comment]
After a bit of wrangling, I don’t think this is limited to Windows.

**Comment by zachleat** [comment]
I did some checking at the commit at https://github.com/11ty/eleventy/commit/8610d473deca4e4e7853e0617764f462ec15b179 does apply and fix 0.7.1 as well. So this will be included with upcoming 0.7.2.
